source/features/bugs-tab.tsx

```diff
@@ -11,6 +11,24 @@ import {getRepo} from '../github-helpers';
 import SearchQuery from '../github-helpers/search-query';
 import abbreviateNumber from '../helpers/abbreviate-number';
 
+const cacheKey = (): string => __filebasename + ':' + getRepo()!.nameWithOwner;
+
+async function highlightBugsTabOnIssuePage(): Promise<void | false> {
+	if (!await cache.get<boolean>(cacheKey())) {
+		return false;
+	}
+
+	const sideBarLables = await elementReady('#partial-discussion-sidebar .sidebar-labels');
+	const hasBugLabel = select.exists('.IssueLabel[href$="/bug" i]', sideBarLables);
```

Comment:
You can merge these. Since _change_ is only needed if **it exists**, waiting until dom-ready is fine. 

Likewise, generally I assume that if an issue is a bug _now_, it will still be a bug when you go _back_ to it, so the  `selected` tab doesn't necessarily need to be updated.

Maybe just:
```js
	if (!await elementReady('.IssueLabel[href$="/bug" i]')) {
		return false
	}
```